packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -267,4 +267,24 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
 
     Terra.validates.element('variants');
   });
+
+  describe('When navigating between first and last items', () => {
+    it('Navigates from first item to last item', () => {
+      browser.url('/raw/tests/cerner-terra-core-docs/dropdown-button/variant-dropdown-buttons');
+      $('[class*=dropdown-button]').click();
+      $('[class*=dropdown-list]').waitForDisplayed();
+      $('li:first-child[role="menuitem"]').isFocused();
```

Comment:
This line is not doing anything, it is simply returning a boolean without any operation (see [the doc](https://webdriver.io/docs/api/element/isFocused/) for isFocused); can we update this to use an `expect` similar to [this](https://github.com/cerner/terra-core/blob/35cbecb6941c298631f9eabd0dd3fe1795622a55/packages/terra-show-hide/tests/wdio/show-hide-spec.js#L25)?

Reply:
Reverted these changes.